.gitignore

```diff
@@ -1,8 +1,8 @@
 # Node modules
 node_modules/
-package-lock.json
+
+# Lock files
```

Comment:
We don't have commit checker now, but can you rename your commit  with:
"feat: migrate project to use pnpm"